### PR TITLE
Replace oasis-node with Oasis CLI

### LINKED ISSUE #394: Replace oasis-node docs with CLI for token management
Revamp https://docs.oasis.io/general/manage-tokens/advanced/oasis-cli-tools/ to use CLI instead of `oasis-node` for most operations.

### PR DESCRIPTION
**Pull request by matevz**
Fixes #394.

Requires https://github.com/oasisprotocol/cli/pull/41

Preview of the [new CLI chapter](https://deploy-preview-399--trusting-archimedes-14c863.netlify.app/general/manage-tokens/cli/).

### CODE REVIEW

File: docs/general/manage-tokens/cli/wallet.md [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+lester                          ledger (ed25519-legacy:0)       oasis1qzdyu09x7hs5nqa0sjgy5jtmz3j5f99ccq0aezjk
+$ oasis wallet rm lester
+WARNING: Removing the account will ERASE secret key material!
+WARNING: THIS ACTION IS IRREVERSIBLE!
+? Enter 'I really want to remove account lester' (without quotes) to confirm removal: I really want to remove account lester
+$ oasis wallet list
+ACCOUNT                         KIND                            ADDRESS                                        
+```
+
+## Remote Signer for `oasis-node`
```
Comment (kostko) [inline docs/general/manage-tokens/cli/wallet.md]: I would move this to some advanced section.

File: docs/general/manage-tokens/cli/wallet.md [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+Node Args:
+  --signer.backend=remote \
+  --signer.remote.address=unix:/datadir/oasis-oscar.socket
+
+*** REMOTE SIGNER READY ***
+```
+
+## Testing wallet
+
+Oasis CLI comes with the following hardcoded testing accounts:
```
Comment (kostko) [inline docs/general/manage-tokens/cli/wallet.md]: Add warning that private keys for these accounts are well-known and that using them on public networks will result in the funds being drained.

File: docs/general/manage-tokens/staking-and-delegating.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ The Oasis Network is supported by an amazing community of validators and infrast
 
 ### Tools to use for Staking and Delegation
 
-In addition to using the [Oasis CLI](advanced/oasis-cli-tools/README.md) for generating the `staking.AddEscrow` transaction, there is a number of wallets and custodians that support staking and delegation on the Oasis Network. Those include:
+In addition to using the [Oasis CLI](cli/README.md) for generating the `staking.AddEscrow` transaction, there is a number of wallets and custodians that support staking and delegation on the Oasis Network. Those include:
```
Comment (kostko) [inline docs/general/manage-tokens/staking-and-delegating.md]:
```suggestion
In addition to using the [Oasis CLI](cli/README.md) for generating the `staking.AddEscrow` transaction, there are a number of wallets and custodians that support staking and delegation on the Oasis Network. Those include:
```

File: docs/general/manage-tokens/terminology.md [hunk 4]
```diff
@@ -66,8 +66,9 @@ if we manually deposited them).
 
 ## Slashing
 
-Your delegated tokens can be slashed if the node that you delegated your tokens
-to gets slashed for double signing.
+A portion of your delegated tokens can be **slashed** (seized) by the network,
+if the node that you delegated your tokens to gets slashed, e.g. as a penalty
+for validating invalid blocks containing double signed transactions.
```
Comment (kostko) [inline docs/general/manage-tokens/terminology.md]:
```suggestion
for equivocating in the protocol by signing diverging blocks for the same height.
```

File: docs/node/run-your-node/prerequisites/stake-requirements.md [hunk 5]
```diff
@@ -22,14 +22,13 @@ such as [Oasis Scan].
 For more information about obtaining information on your entity's account, see
 the [Account Get Info] doc.
 
-Staking thresholds may change in the future. Read [Common staking info] chapter,
-to learn how to check the current network thresholds directly from the
-`oasis-node`.
+Staking thresholds may change in the future. Read the
+[native token information] to see the current values of the network.
```
Comment (kostko) [inline docs/node/run-your-node/prerequisites/stake-requirements.md]:
```suggestion
[native token information] to see the current values used by the network.
```

File: docs/node/run-your-node/validator-node.mdx
Comment (kostko) [inline docs/node/run-your-node/validator-node.mdx:1]: This guide is still full of stuff that is made obsolete by the CLI (e.g. updating entity descriptors, checking node status, delegating to your entity) and we should ideally change these as well. We may want to do this in a follow-up cleanup PR though.

File: docs/general/manage-tokens/terminology.md (hunk 4 above)
Comment (matevz) [reply docs/general/manage-tokens/terminology.md]: _equivocating_, interesting term.

File: docs/node/run-your-node/validator-node.mdx
Comment (matevz) [reply docs/node/run-your-node/validator-node.mdx:1]: These are all in the "Run Node" chapter. Let's do this in a separate PR. Opened issue https://github.com/oasisprotocol/docs/issues/505

**Review by kostko — APPROVED** [review]

**Pull request merged**